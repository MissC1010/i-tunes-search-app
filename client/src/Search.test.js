import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import React from "react";
import Search from "./SearchForm";

test("renders search form", () => {
  render(<Search />);
  expect(
    screen.getByPlaceholderText("Search for content...")
  ).toBeInTheDocument();

  expect(screen.getByText("Search")).toBeInTheDocument();
});

test("snapshot", () => {
  const { asFragment } = render(<Search />);
  expect(asFragment()).toMatchSnapshot();
});
