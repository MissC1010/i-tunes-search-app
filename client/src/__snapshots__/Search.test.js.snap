// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<DocumentFragment>
  <form
    class="search-form"
  >
    <div
      class="search-input input-group"
    >
      <input
        class="search-text form-control"
        placeholder="Search for content..."
        type="text"
        value=""
      />
      <select
        class="custom-select form-select"
      >
        <option
          value="movie"
        >
          Movie
        </option>
        <option
          value="podcast"
        >
          Podcast
        </option>
        <option
          value="music"
        >
          Music
        </option>
        <option
          value="musicVideo"
        >
          Music Video
        </option>
        <option
          value="audiobook"
        >
          Audiobook
        </option>
        <option
          value="shortFilm"
        >
          Short Film
        </option>
        <option
          value="tvShow"
        >
          TV Show
        </option>
        <option
          value="software"
        >
          Software
        </option>
        <option
          value="ebook"
        >
          E-book
        </option>
        <option
          selected=""
          value="all"
        >
          All
        </option>
      </select>
      <button
        class="search-button btn btn-outline-secondary"
        type="submit"
      >
        Search
      </button>
    </div>
  </form>
</DocumentFragment>
`;
